ceph.spec.in

```diff
@@ -14,7 +14,7 @@
 #
 # Please submit bugfixes or comments via http://tracker.ceph.com/
 # 
-%bcond_with ocf
+%bcond_without ocf
```

Comment:
very cool! thank you @smithfarm ! now i understand why we also package selinux now =D